06-Customizing AzTS for your org/Extending AzTS/ManagementGroupBasedComplianceInitiative.md

```diff
@@ -67,6 +67,34 @@ By default, this feature is not turned on. If you haven't enabled it in your AzT
 10. Click 'OK' and then click 'Save'.
 ![Resources](../../Images/06_MGCIEnableMGAndSave.png)
 
+-----
+
+- Using helper script
+  - Download the script from [here](../../TemplateFiles/DeploymentFiles.zip), skip this step if you have already downloaded the deployment package zip
+  > **Note:** Script can be downloaded by clicking Alt+Raw button.
+  - Open a PowerShell session.
+  - Navigate to the download location of the script in PowerShell session.
+    - cd "Script download location" 
+
+  - Unblock the downloaded script.
+    - Unblock-File -Path ".\ConfigureAzTSFeature.ps1"
+  - Load the script in current PowerShell session.
+    - . ".\ConfigureAzTSFeature.ps1"
+    > **Note:** Do not miss the '.' at beginning of the above command.
+  - Connect to AzAccount
+    - Connect-AzAccount -Tenant "AzTSHostingTenantId"
```

Comment:
Use <AzTSHostingTenantId> instead of just AzTSHostingTenantId

Reply:
Done